src/node.cc

```diff
@@ -4425,15 +4425,22 @@ int EmitExit(Environment* env) {
 }
 
 
-IsolateData* CreateIsolateData(Isolate* isolate, uv_loop_t* loop) {
-  return new IsolateData(isolate, loop, nullptr);
+ArrayBufferAllocatorBase* CreateArrayBufferAllocator() {
```

Comment:
I think we could also use a `std::unique_ptr`, and/or drop `FreeArrayBufferAllocator` because that would be implicit in using `std::unique_ptr`.

Reply:
I just followed the pattern of the existing APIs like FreeIsolateData, FreeEnvironment and FreePlatform.  I think they were made in the current pattern to avoid trouble of [unique_ptr for ABI](https://stackoverflow.com/questions/24656996/abi-in-pimpl-idiom-with-unique-ptr). 
Do you think I can keep this pattern for ArrayBufferAllocator?